app/actions/AccountActions.js

```diff
@@ -25,12 +25,18 @@ class AccountActions {
         return dispatch => {
             if (!accountSearch[uid]) {
                 accountSearch[uid] = true;
-                return AccountApi.lookupAccounts(start_symbol, limit).then(
-                    result => {
+                return AccountApi.lookupAccounts(start_symbol, limit)
+                    .then(result => {
                         accountSearch[uid] = false;
                         dispatch({accounts: result, searchTerm: start_symbol});
-                    }
-                );
+                    })
+                    .catch(error => {
+                        console.log(
+                            "We have error with accountSearch mathod: ",
+                            error
+                        );
+                        reject(error);
```

Comment:
There is no reject method in this scope.

This is an indirect search. Consider for example AccountNameInput class
https://github.com/blockchainprojects/bitshares-ui/blob/c5016f8a0b15d5b0dedc51b1d69ad4a07f1a8018/app/components/Forms/AccountNameInput.jsx#L11

It triggers the account search via the action
https://github.com/blockchainprojects/bitshares-ui/blob/c5016f8a0b15d5b0dedc51b1d69ad4a07f1a8018/app/components/Forms/AccountNameInput.jsx#L124
This return value of this action is not caught, but dispatched back into the component via AccountStore (this is the alt store mechnism) once the AccountActions.accountSearch Promise has resolved
https://github.com/blockchainprojects/bitshares-ui/blob/c5016f8a0b15d5b0dedc51b1d69ad4a07f1a8018/app/components/Forms/AccountNameInput.jsx#L187-L191

For the example above, please change it that there is a console error call with a precise message, e.g. in the style of
`console.error("Exception in AccountActions.accountSearch(", start_symbol, ", ", limit, " occured: ", err)`

Please check all similar occurrences in introduced ".catch" methods for Promises throughout this PR